styles.css

```diff
@@ -3,7 +3,6 @@ body {
   width: 20.4375rem;
```

Comment:
[OPTIONAL]

Because of this fixed width, our design is not responsive enough in order to accommodate different types of mobiles and tablets screen widths.

- [ ] Please consider making your design adaptable for different screen widths;
(without using media queries for now, but maybe just removing this fixed width, and if needed, add max-width to different sections of your portfolio) :1st_place_medal: